impress-org/givewp
# Feature: Add campaign setting for default form

Resolves [GIVE-1493]

## Description



This PR adds the Default Form setting in the Campaign Details Page, on the Settings tab.

## Affects

- Adds `CampaignDetailsData` DTO class
- Updates response data to use DTO in `CampaignRequestController`

## Visuals



![image](https://github.com/user-attachments/assets/f394ccb7-57b6-4591-b1d4-0dc847aefac7)

## Testing Instructions



Go to the Campaign Details Page
Go to the Settings tab
Select a default form
Publish the campaign
Check that the default form was updated on the Default Form widget

## Pre-review Checklist






[GIVE-1493]: https://stellarwp.atlassian.net/browse/GIVE-1493?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
